### ISSUE TITLE
VSTS Free account users

### ISSUE THREAD

**Issue by Nathan-Tarin** [MicrosoftDocs/visualstudio-docs#795]
Hello,

Can you please confirm how many users do the subscribers get in the VSTS account? I know that they get Basic level accounts, that provide 5 free users, however, I was under the impression that if the benefit comes from a Cloud subscription, instead of a Standard subscription, you get unlimited users on the account. Could you please verify this?

Thanks in advance!

---
#### Document Details

⚠ *Do not edit this section. It is required for docs.microsoft.com ➟ GitHub issue linking.*

* ID: 34c186b2-096e-e066-ed2c-6ae725c5eff1
* Version Independent ID: b93a5dea-8d1f-eb84-0248-5b1fbf2f7638
* Content: [VSTS benefits for Visual Studio subscribers](https://docs.microsoft.com/en-us/visualstudio/subscriptions/vs-vsts)
* Content Source: [subscriptions/vs-vsts.md](https://github.com/Microsoft/visualstudio-docs/blob/master/subscriptions/vs-vsts.md)
* Service: **unspecified**
* GitHub Login: @evanwindom
* Microsoft Alias: **jaunger**

**Comment by TerryGLee** [comment]
@Nathan-Tarin Thanks for your inquiry and apologies for the late reply.
@evanwindom @prrana Is this something that you could help the customer with?

**Comment by AraHaan** [comment]
It would be nice if github users could use their github accounts to log into VSTS as well. :thinking: Or even their microsoft accounts to log into github (like only if their microsoft account is linked to their github). Just a thought I am just placing here for now.

**Comment by evanwindom** [comment]
@Nathan-Tarin   Thanks for your question.  Here's some info from the Billing FAQ that answers your question:
Q: What kind of volume discounts are available for Visual Studio subscriptions?
A: You get a 5% discount on the 6th and all subsequent subscriptions within each type of subscription:
•Visual Studio Professional monthly
•Visual Studio Professional annual
•Visual Studio Enterprise monthly
•Visual Studio Enterprise annual
Also, the discount only applies to charges in a given monthly billing period. So if you buy 5 Visual Studio Professional annual subscriptions in one month, and then you buy 5 more the next month, you'll pay the regular price on all 10 subscriptions.

So, in short, the first five users are included with cloud subscriptions.

**Comment by evanwindom** [comment]
@AraHaan Thank you for your suggestion.  I have forwarded your comment to the engineering team for their consideration.

**Comment by jongdaelee** [comment]
So to confirm and clarify, "So, in short, the first five users are included with cloud subscriptions." means I only get 5 free access to VSTS just like anyone else who does not have paid MSDN subscription?

**Comment by evanwindom** [comment]
@jongdaelee Thanks for your question.  I think my initial response may have caused some confusion.  There are volume discounts for cloud subscriptions, and a tiered pricing model for VSTS.  To answer your question directly, the number of users is the same, but the capabilities of VSTS in the paid subscriptions can vary from the free accounts.  Visual Studio Enterprise standard, as well as monthly and annual cloud subscriptions include Test Manager, Package Management and Private Pipelines, which are not included in free accounts.  For more information on what's included in the subscription levels, please visit  https://visualstudio.microsoft.com/vs/pricing/.
